cf-agent/verify_packages.c

```diff
@@ -2706,16 +2708,18 @@ static bool ExecuteSchedule(EvalContext *ctx, const PackageManager *schedule, Pa
                     EvalContextStackPushPromiseFrame(ctx, ppi);
                     if (EvalContextStackPushPromiseIterationFrame(ctx, NULL))
                     {
-                        if (ExecPackageCommand(ctx, command_string, verify, true, &a, ppi, &result))
+                        bool ok = ExecPackageCommand(ctx, command_string, verify, true, &a, ppi, &result);
+
+                        if (IGNORE_THIS_PACKAGE(pi->name))
```

Comment:
Use `StringSafeEqual()` instead of the macro with strcmp:

```C
if (StringSafeEqual(pi->name, PACKAGE_IGNORED_CFE_INTERNAL))
{
```

Reply:
Thanks, I pushed an updated patch that uses StringSafeEquals instead of the macro.